<launch>
	<!-- This node will launch MAVROS -->
	<include file="$(find mavros)/launch/px4.launch"/>
	<rosparam param="mavros/system_id">1</rosparam>
	<rosparam param="mavros/component_id">240</rosparam>
	<!--
	<rosparam param="mavros/fcu_url">/dev/ttyACM0:115200</rosparam>
	-->
	<rosparam param="mavros/fcu_url">"udp://127.0.0.1:14551@14555"</rosparam>
	<!-- This node will run  run.cpp and server.py -->
	<node name="drone_node" pkg="ciis_drone" type="drone_node" output="screen" />
	<!-- add delay waiting server to turn on -->
	
	<arg name="node_start_delay" default="30.0" />  
	<node name="model_node" pkg="ciis_drone" type="controller.py"  output="screen"/>

</launch>